riva/clients/nmt/riva_streaming_s2s_client.cc

```diff
@@ -70,6 +70,10 @@ DEFINE_bool(
     "True returns text exactly as it was said with no normalization.  False applies text inverse "
     "normalization");
 DEFINE_string(ssl_cert, "", "Path to SSL client certificatates file");
+DEFINE_string(tts_encoding, "", "TTS output encoding, currently either PCM or OPUS");
```

Comment:
may be set default encoding to PCM ?

Reply:
we are already setting it to pcm if the string is empty, so this case is taken care. Empty and pcm behave same.

Reply:
right. but if you set a default value here, you don't need to check for empty().
its fine either way.